### DISCUSSION TITLE
Tailwind in Wordpress Plugins? Custom Blocks?

### DISCUSSION THREAD

**Discussion by justinluk** [gregsullivan/_tw#35] [General]
Hi there,

I'm working on building some custom blocks to be used in the Gutenberg editor. Is there a way to get Tailwind classes from this theme to be used within my custom blocks?

Thanks!
Justin

**Comment by justinluk** [comment]
From what I can tell, the `style-editor.css` isn't being loading in the admin area.

**Comment by gregsullivan** [comment]
Hi Justin!

Thanks for posting this, and sorry for the delay in responding—I'm just now getting caught up after the holidays.

I wrote this in response to an earlier issue on the same theme as your question:

https://github.com/gregsullivan/_tw/issues/25#issuecomment-1253941338

Does that help, or lead to any new questions?

The `style-editor.css` file is loaded by the block editor, modified, and then injected into a `style` element, so it won't appear in the network panel. (And it will only be loaded by the block editor itself, not on other admin pages.)

Please let me know if you have any other questions!

Greg
